metrics/web/test/common/presentation/routes/metrics_page_route_test.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:metrics/common/presentation/routes/metrics_page_route.dart';
+import 'package:metrics/common/presentation/routes/route_name.dart';
+import 'package:test/test.dart';
+
+import '../../../test_utils/matcher_util.dart';
+
+void main() {
+  group('MetricsPageRoute', () {
```

Comment:
We should use the double-quoted strings for group names and test descriptions according to the [dart code style](https://github.com/platform-platform/monorepo/blob/master/docs/10_dart_code_style.md#test-descriptions) document. Also, maybe we can check that the `buildTransitions` function returns the given child to be sure that there are no animations?